gapic-generator/lib/gapic/schema/api.rb

```diff
@@ -397,6 +415,9 @@ def sanity_checks output
           output.puts "WARNING: configured common service #{k} is not present" unless addrs.include? k
           output.puts "WARNING: configured common service delegate #{v} is not present" unless addrs.include? v
         end
+        # TODO: Sanity check snippet configs to ensure that the type of call
```

Comment:
It's fine if you want to have this check here, but in principle, this will be done on the config repo side of things.